test/generators/install_generator_test.rb

```diff
@@ -119,4 +119,23 @@ class InstallGeneratorTest < Rails::Generators::TestCase
       assert_match 'config.allow_cookie_authentication = true', shopify_app
     end
   end
+
+  test "generates javascript assets required when embedded" do
+    ShopifyApp.stubs(:use_webpacker?).returns(false)
+
+    run_generator %w(--embedded)
+    assert_file "app/assets/javascripts/shopify_app.js"
+    assert_file "app/assets/javascripts/flash_messages.js"
+  end
```

Comment:
I am currently missing key understanding into how apps behave when they use Turbolinks but don't rely on Webpack. How do we let apps opt-in to the `turbolink.js` asset if they don't use Webpack?